tasks/msapprovals/msapprovals.go

```diff
@@ -247,7 +247,9 @@ func (p *Task) getTransactionIfApplied(ctx context.Context, msg *types.Message,
 		}
 
 		// Get state of actor before the message was applied
-		act, err := p.node.StateGetActor(ctx, msg.To, pts.Key())
+		// pts is the tipset containing the messages, so we need the state as seen at the start of message processing
+		// for that tipset
+		act, err := p.node.StateGetActor(ctx, msg.To, pts.Parents())
```

Comment:
Did some underlying behaviour change that now we need to use the parents?

Reply:
No, but pts is the tipset containing the message, so we need to look at the actor state before the message was applied